KUBEVIRT_LOCAL_TESTING.md

```diff
@@ -0,0 +1,40 @@
+# [kubevirtci](README.md): Testing kubevirt with locally provisioned cluster
+
+With the changes in place you also can directly execute [`make functest`](https://github.com/kubevirt/kubevirt/blob/master/docs/getting-started.md#testing) against a cluster locally with kubevirt that was provisioned using `kubevirtci`.
+
+Steps:
+
+### kubevirtci: provision cluster locally
+
+```bash
+cd .../kubevirtci
+cd cluster-provision/k8s/1.17.0
+./provision.sh
```

Comment:
Add note that provision.sh also runs up.sh so it also checks that cluster-up is working

Also that cluster-up/... script could need tunning for new k8s versions.

Reply:
Done.

Reply:
I didn't address the second comment, because that should be an issue IMHO.